docs/scripting.md

```diff
@@ -278,7 +278,7 @@ module.exports = (robot) ->
   )
 ```
 
-See [the design patterns document](patterns.md) for examples of complex matchers.
+See [the design patterns document](https://hubot.github.com/docs/patterns/#dynamic-matching-of-messages) for examples of complex matchers.
```

Comment:
I think this should be `patterns.md#dynamic-matching-of-messages`. Jekyll will rewrite it on hubot.github.com.